src/pages/page4/page4.html

```diff
@@ -19,18 +18,10 @@
       <ion-label floating>access token</ion-label>
       <ion-input type="text" required="" value="{{access_token}}" readonly></ion-input>
     </ion-item>
+  </ion-item-group>
 
-    <ion-item>
-      Time Zone
-      <ion-select>
-        <ion-option value="eastern" selected="true">Eastern</ion-option>
-        <ion-option value="central">Central</ion-option>
-        <ion-option value="mountain">Mountain</ion-option>
-        <ion-option value="pacific">Pacific</ion-option>
-      </ion-select>
-    </ion-item>
-
-    <ion-list-header>Notifications</ion-list-header>
+  <ion-item-group>
+    <ion-item-divider color="light">Notifications</ion-item-divider>
```

Comment:
The divider looks really weird here. Maybe it should be before the Notifications text.